### ISSUE TITLE
MiniZinc 2.5.0 COIN Defaults

### ISSUE THREAD

**Issue by klindek** [MiniZinc/MiniZincIDE#112]
Getting these error messages based on options for the COIN solver. I created a new project just in case it was my project settings causing the issues. Upgraded from Previous MiniZinc, win64, Windows 10.

With Solver Statistics Enabled:
minizinc: Unrecognized option or bad format `--solving-statistics'

N>2 (although parallel COIN has never worked as far as I know)
minizinc: Unrecognized option or bad format `--parallel'

**Comment by klindek** [comment]
Sorry, should be "COIN Options" not "COIN Defaults"

The defaults are fine.

**Comment by cyderize** [comment]
Yes, it appears that the IDE is using the an invalid option when choosing solving statistics. I'll fix this in the next patch release.

**Comment by klindek** [comment]
Is there a short term solution? I use the stats to see when I've really screwed things up.

**Comment by cyderize** [comment]
You can add a custom parameter in the 'Extra configurations parameters' area with the name `-s` and the type `Boolean` set to the value `true`. That should pass the full statistics option.

**Comment by cyderize** [comment]
The Windows build of CBC does not include parallelism support (the problem is that MiniZinc and IDE don't know that, so the option is still there). Other platforms do support it, so if you have access to a linux/macOS machine it should work.

We may be able to get a parallel-enabled build of CBC working on Windows at some point and then that option will function correctly (although parallel CBC generally doesn't seem to perform any better than the single-threaded one).

**Comment by zayenz** [comment]
The same problem also exists for the Gecode solver, and it unfortunately makes 2.5.0 unusable for now, since I also depend on the statistics when developing models. 

Testing the work-around by adding an extra configuration parameter, I tried 
* adding `--solver-statistics` or `-s` with Boolean true (this did not output any additional statistics)
* adding `--solver-statistics` or `-s`  with String and empty, errored with `minizinc: Unrecognized option or bad format ''`

I also tried with `solver-statistics` with the same problems.

At first, I added a parameter and entered `-s` into the Value field since that is where focus was, and I did not really see that there was a field below Parameter. This gave the error `minizinc: Unrecognized option or bad format '--'`. I got a similar error when trying with just `s`: `minizinc: Unrecognized option or bad format '--s'`.
